content/en/serverless/installation/dotnet.md

```diff
@@ -91,28 +92,34 @@ More information and additional parameters can be found in [Datadog Serverless C
 {{% /tab %}}
 {{% tab "Serverless Framework" %}}
 
-1. Add the following layers and environment variables to each .NET Lambda function you wish to instrument:
+The [Datadog Serverless Plugin][1] automatically adds the Datadog Lambda Library to your functions using Lambda Layers, and configures your functions to send metrics, traces, and logs to Datadog through the [Datadog Lambda Extension][2].
```

Comment:
I don't think this is accurate for .NET -- we don't add the Lambda Library for the .NET runtime